parlai/mturk/core/agents.py

```diff
@@ -345,6 +363,13 @@ def pay_bonus(self, bonus_amount, reason='unspecified'):
         if self.manager.pay_bonus(worker_id=self.worker_id, bonus_amount=bonus_amount, assignment_id=self.assignment_id, reason=reason, unique_request_token=unique_request_token):
             print("Paid $" + str(bonus_amount) + " bonus to WorkerId: " + self.worker_id)
 
+    def email_worker(self, subject, message_text):
+        response = self.manager.email_worker(worker_id=self.worker_id, subject=subject, message_text=message_text)
+        if 'success' in response:
+            print("Email sent to worker ID: "+str(self.worker_id)+": Subject: "+str(subject)+": Text: "+str(message_text))
+        elif 'failure' in response:
+            print("Unable to send email to worker ID: "+str(self.worker_id)+". Error: "+str(response['failure']))
+
```

Comment:
great catch - I added a return value to the method and leave it to the user to decide how to handle the error